include/oneapi/dpl/pstl/hetero/numeric_impl_hetero.h

```diff
@@ -152,13 +152,27 @@ __pattern_transform_scan_base_impl(_ExecutionPolicy&& __exec, _Iterator1 __first
     }
 }
 
+template <typename _Iterator1, typename _Iterator2, typename = void>
+struct is_equal_exist : std::false_type
```

Comment:
What is a goal of `is_equal_exist`?
Also name is not so good with self-documented naming perspective.